pkg/agent/sysadvisor/metacache/metacache.go

```diff
@@ -403,6 +406,14 @@ func (mc *MetaCacheImp) storeState() error {
 	checkpoint.PodEntries = mc.podEntries
 	checkpoint.PoolEntries = mc.poolEntries
 
+	begin := time.Now()
```

Comment:
Do we need to log error when storing state exceeds time limit in any case? If so, move this hook function to store/restore state in metacache.

Reply:
Yes, this hook is already in storeState, and also called by restore

Reply:
In storeState, exceeding time limit is not checked.